cegs_portal/search/templates/search/v1/search_results.html

```diff
@@ -251,21 +382,28 @@
         import { getJson } from "{% static 'search/js/files.js' %}";
         import { State } from "{% static 'search/js/state.js' %}";
         import { downloadRegionsSetup } from "{% static 'search/js/search_results/downloads.js' %}";
-        import { STATE_DHS_EFFECT, STATE_FACETS, STATE_REGION } from "{% static 'search/js/search_results/consts.js' %}";
+        import { STATE_DHS_EFFECT, STATE_FACETS, STATE_REGION, STATE_FEATURE_TYPES } from "{% static 'search/js/search_results/consts.js' %}";
 
         let searchURL = function(facets) {
             let enabledFacets = facets.map(id => `facet=${id}`)
                                   .join('&')
             return `{% url 'search:results' %}?query={{ query }}${enabledFacets !== '' ? '&' + enabledFacets : ''}`;
         }
 
+        let searchFeatureTypeURL = function(feature_types) {
```

Comment:
This isn't used at all. but also, it should be merged with `searchURL`. The purpose of `searchURL` is to ensure that the query parameters are updated to match the selected facets. Now that we are selected feature types `searchURL` should also ensure that query parameters are updated to match selected facet types as well.